tron/serialize/runstate/dynamodb_state_store.py

```diff
@@ -30,18 +28,13 @@ def restore(self, keys) -> dict:
         Fetch all under the same parition key(keys).
         ret: <dict of key to states>
         """
-        translated_items = {}
         try:
             first_items = self._get_first_partitions(keys)
             remaining_items = self._get_remaining_partitions(first_items)
             items = self._merge_items(first_items, remaining_items)
-            #TODO: remove this after berkleyDB is removed.
-            for key in keys:
-                if str(key) in items:
-                    translated_items[key] = items[str(key)]
         except Exception as e:
             self.alert(str(e))
```

Comment:
Please log this too. Also, what should happen if Tron is unable to restore the state of a job when starting up? It looks like we would return empty items, which would cause Tron to start overwriting the state for all jobs with empty state. We definitely don't want that, try to handle it so the impact is as small as possible.

This didn't affect us before when we were relying on the state from bdb, but it will once we stop using bdb.

Reply:
I think restore should not and cannot restore anything if restore failed. If that ever happens, shelvestore could be used as a backup? Or we can manually restore dynamodb states from backup.